### PR TITLE
Include `state` in all events

### PR DESCRIPTION
**Pull request by louischan-oursky**
ref #3443

### CODE REVIEW

File: pkg/auth/webapp/ui_param.go [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
 	"github.com/authgear/authgear-server/pkg/util/intl"
 )
 
-// ClientIDCookieDef is a HTTP session cookie.
-var ClientIDCookieDef = &httputil.CookieDef{
-	NameSuffix: "client_id",
-	Path:       "/",
-	SameSite:   http.SameSiteNoneMode,
-}
+// ClientIDCookieDef is deprecated.
```
Comment (tung2744) [inline pkg/auth/webapp/ui_param.go:13]: Why don't we simply remove it? 🤔

Comment (louischan-oursky) [reply pkg/auth/webapp/ui_param.go:13]: Leaving the lines here to prevent us from re-adding it in the future. Also for easier `git blame`

**Comment by tung2744** [comment]
@louischan-oursky Conflicted, please rebase :adore:

**Comment by louischan-oursky** [comment]
Rebased.

**Pull request merged**